### PR TITLE
Unicode Emoji Support

### PR DESCRIPTION
**Pull request by haukened**
# Unicode Emoji Support!
This PR created the unicode representations of emojis that @C0DK requested instead of short name descriptions.

## Features:

1. Config file support
1. Represents most emojis as Unicode.  Certain compound emojis that require a non-space joiner cannot be viewed on the command line.
1. Works through PuTTY and other SSH/MOSH clients

## Requirements

1. System must have emoji fonts installed.  Windows 10, Ubuntu 18.04 LTS tested with emojiOne fonts working.

## Configuration
kbtui.toml need to have [Emoji] section added with unicodeSupport set to true
example:
```
[Emojis]
unicodeSupport = true
```

## Major Changes

 - Hooked into `printToView()` to call `emojiUnicodeConvert()`
 - Hooked into `handleInput()` to call `resolveRootEmoji()`

### CODE REVIEW

**Comment by haukened** [comment]
After discussing with @kf5grd we should hold these PR's until his config work is merged, then fix anything that's broken. I think this will prevent re-work.

File: .gitignore [hunk 1]
```diff
@@ -1,8 +1,10 @@
 kbtui
 emojiList.go
+kbtui.toml
```
Comment (C0DK) [inline .gitignore]: You should probably remove this

**Comment by Rudi9719** [comment]
Please resolve your merge conflicts and I'll pull it in! I look forward to the emoji support!

File: .gitignore (hunk 1 above)
Comment (haukened) [reply .gitignore]: Resolved - i don't usually see config files included in git, but it seems to be what people want in favor of a defconfig!

**Comment by haukened** [comment]
resolved merge conflicts and removed cruft from `emojiMap.go` - builds on my machine.  If travis passes us we should be good to merge!

enjoy your eggplants!

**Pull request merged**